Snakefile

```diff
@@ -13,19 +13,30 @@ rule all:
     input:
         expand("{output_dir}/cna.txt", output_dir=output_dir)
 
+## Function that checks if the docker: booleaon in config is true, and if so, don't require pdt_installed.txt
+def check_docker():
```

Comment:
For python style, you should put the comment above, in a multi-line string - just below the def.  Something like:
```
"""Ploidetect installed filepath.

Filename to create on a successful install or check for successful installation. 
Checks the config file for docker options.
"""
```

Reply:
Committed the change